Double-comment results in Uncaught SyntaxError: Unexpected token '/'

# 🐛 bug report

A comment line of `// /** @deprecated */` breaks the development build.

## 🎛 Configuration (.babelrc, package.json, cli command)

https://github.com/cubing/cubing.js/blob/main/package.json

## 🤔 Expected Behavior

Parcel should successfully build and serve code containing this:

    // /** @deprecated */
    // get nestedUnits(): Unit[] {
    //   warnOnce("deprecated: nestedUnits");
    //   return Array.from(this.#units);
    // }

https://github.com/cubing/cubing.js/blob/235b19ea8c571f01cf15a1908fb4323684d06c5b/src/cubing/alg/Alg.ts#L137-L141

## 😯 Current Behavior

```
Uncaught SyntaxError: Unexpected token '/'
index.b94739d4.js:61 Uncaught Error: Cannot find module '5Re8D'
    at newRequire (index.b94739d4.js:61)
    at newRequire (index.b94739d4.js:45)
    at localRequire (index.b94739d4.js:83)
    at Object.15Gne.three (index.ts:3)
    at newRequire (index.b94739d4.js:71)
    at localRequire (index.b94739d4.js:83)
    at Object.7il72../app (HMRRuntime.js:366)
    at newRequire (index.b94739d4.js:71)
    at index.b94739d4.js:120
    at index.b94739d4.js:143
newRequire @ index.b94739d4.js:61
newRequire @ index.b94739d4.js:45
localRequire @ index.b94739d4.js:83
15Gne.three @ index.ts:3
newRequire @ index.b94739d4.js:71
localRequire @ index.b94739d4.js:83
7il72../app @ HMRRuntime.js:366
newRequire @ index.b94739d4.js:71
(anonymous) @ index.b94739d4.js:120
(anonymous) @ index.b94739d4.js:143
```

## 💁 Possible Solution

Improve handling of comments?

We can easily work around this by removing or modifying the double-commented code. But this error threw me off for a bit, because it resembles [the `Unexpected token '<'` error](https://github.com/parcel-bundler/parcel/issues/2855) I see on a daily basis. It would be nice if a valid JS comment doesn't break the build, since the error resembles the kind of non-semantic code transformation that can lead to security bugs through injection.

## 🔦 Context

I'm trying to upgrade to the latest Parcel nightly for our project.

## 💻 Code Sample

```shell
git clone https://github.com/cubing/cubing.js
cd cubing.js
git checkout 235b19ea8c571f01cf15a1908fb4323684d06c5b
npm install
make dev
# Visit e.g. http://localhost:3333/twizzle/index.html?puzzle=3x3x3
```

I'm sorry, I also tried to make a simple repro for this and was unable to make one.

I tried to bisect which Parcel version this started with, but nightly versions in the mid-500s run into a `TypeError: command.name is not a function` error that makes it hard to tell when this issue started.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.0.0-nightly.643+3baf990a |
| Node             | v15.12.0 | 
| npm/Yarn         | 7.6.3 |
| Operating System | macOS 11.2.3 |

--- a/packages/shared/babel-ast-utils/src/generator.js
+++ b/packages/shared/babel-ast-utils/src/generator.js
@@ -274,7 +274,7 @@ for (let key in generator) {
     // These are printed by astring itself
     if (node.trailingComments) {
       for (let c of node.trailingComments) {
-        if (c.type === 'CommentLine') {
+        if (c.type === 'CommentLine' || c.type === 'LineComment') {
           c.type = 'LineComment';
         } else {
           c.type = 'BlockComment';

--- a/packages/shared/babel-ast-utils/test/fixtures/class.js
+++ b/packages/shared/babel-ast-utils/test/fixtures/class.js
@@ -7,6 +7,7 @@ class C extends A {
   get property() {
     return this._property;
   }
+  // /* foo */
   set property(value) {
     this._property = value;
   }

